workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/NewAppFlow/constants/index.tsx

```diff
@@ -80,6 +80,13 @@ export const APP_FLOW_ITEMS = [
 	{
 		description: `Inform the support and help references. This will impact how users will experience this app's customer support and learning.`,
 		label: 'Support',
+		parseSchema: (context: NewAppInitialState) => {
+			if (context.pricing.priceModel === 'Paid') {
```

Comment:
Use the enum for priceModel

```suggestion
			if (context.pricing.priceModel === ProductPriceModel.PAID) {
```